railties/lib/rails/generators/rails/app/templates/github/ci.yml.tt

```diff
@@ -125,6 +125,7 @@ jobs:
           <%- elsif options[:database] == "postgresql" -%>
           DATABASE_URL: postgres://postgres:postgres@localhost:5432
           <%- end -%>
+          # RAILS_MASTER_KEY: ${{ secrets.RAILS_TEST_KEY }}
```

Comment:
People who are not using `config/credentials/test.yml.enc` can run tests without `RAILS_MASTER_KEY`.
How about adding an additional comment to say that only those who are using credentials in a test environment need to set it?

```yaml
# If you are using credentials in a test environment, set the value of `config/credentials/test.key`
# RAILS_MASTER_KEY: ${{ secrets.RAILS_TEST_KEY }}
```